src/routing/interfaces.d.ts

```diff
@@ -129,42 +123,37 @@ export interface OnExit {
 	(): void;
 }
 
-/**
- * Outlet options that can be configured
- */
-export interface OutletOptions {
-	key?: RegistryLabel;
-	mapParams?: MapParams;
-}
+export interface MatchDetails {
```

Comment:
Is `OutletDetails` a better name?

Reply:
I think `MatchDetails` is more appropriate as it's the details for the current match, not the outlet as a whole. Perhaps `MatchProperties` though?

Reply:
Don't want to overload `properties`